server/src/test/java/org/apache/iotdb/db/integration/IoTDBDeletionIT.java

```diff
@@ -73,6 +74,73 @@ public void tearDown() throws Exception {
     IoTDBDescriptor.getInstance().getConfig().setPartitionInterval(prevPartitionInterval);
   }
 
+  /**
+   * Should delete this case after the deletion value filter feature be implemented
+   *
+   * @throws SQLException
+   */
+  @Test
+  public void testUnsupportedValueFilter() throws SQLException {
+    try (Connection connection =
+            DriverManager.getConnection(
+                Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+
+      statement.execute("insert into root.vehicle.d0(time,s0) values (10,310)");
+      statement.execute("insert into root.vehicle.d0(time,s3) values (10,'text')");
+      statement.execute("insert into root.vehicle.d0(time,s4) values (10,true)");
+
+      String errorMsg =
+          "303: Check metadata error: For delete statement, where clause can only"
+              + " contain time expressions, value filter is not currently supported.";
+
+      try {
+        statement.execute("DELETE FROM root.vehicle.d0.s0  WHERE s0 <= 300 AND s0 > 0");
```

Comment:
hi, add a test case,  DELETE FROM root.vehicle.d0.s0  WHERE s0 <= 300 and time > *** and time < ***.
what will this situation return ?

Reply:
It will return"For delete statement, where clause can only contain atomic expressions like : time > XXX, time <= XXX, or two atomic expressions connected by 'AND'"

In situations expressions number more than 2, will always return this message.

Related code:

> List<FilterOperator> children = filterOperator.getChildren();  
    FilterOperator lOperator = children.get(0);
    FilterOperator rOperator = children.get(1);
    if (!lOperator.isLeaf() || !rOperator.isLeaf()) {
      throw new SQLParserException(DELETE_RANGE_ERROR_MSG);
    }